bots/hermes/README.md

```diff
@@ -1,3 +1,25 @@
-Requirements: 
-Python (3.8.10 recommended)
-pip (21.1.2)
+# hermes_bot
+
+## Requirements
+
+- Python (3.x.x recommended)
+- pip (21.1.2)
+- Bearer Token from @bot father intelegram 
+
+### Setting up a python virtualenv
+
+- Set up your virtualenv and install requirements: 
+```shell
+make install
+```
+
```

Comment:
Since we changed the Makefile I guess this step became irrelevant. We can replace it with a `make run` instead